ansible_events_ui/api/project.py

```diff
@@ -0,0 +1,125 @@
+import sqlalchemy as sa
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from ansible_events_ui import schemas
+from ansible_events_ui.db import models
+from ansible_events_ui.db.dependency import get_db_session
+from ansible_events_ui.project import clone_project, sync_project
+
+router = APIRouter()
+
+
+@router.get("/api/projects/")
+async def list_projects(db: AsyncSession = Depends(get_db_session)):
+    query = sa.select(
+        models.projects.c.id, models.projects.c.url, models.projects.c.name
+    )
+    result = await db.execute(query)
+    return result.all()
+
+
+@router.post("/api/projects/")
+async def create_project(
+    p: schemas.ProjectCreate, db: AsyncSession = Depends(get_db_session)
```

Comment:
Minor: Since this endpoint is being refactored, it'd be nice to use rename single character variable names into something more meaningful (e.g. `project` or `data`).

Reply:
Good idea it will make it much easier to read